packages/emergency-response/src/components/FmaMap/index.js

```diff
@@ -1,104 +1,61 @@
-  openPanel(e) {
-    const layer = e.target;
-    const fmaProperties = layer.feature.properties;
-    this.props.renderPanel(fmaProperties);
-  }
-
-  setOpacity(e) {
-    const layer = e.target;
-    layer.setStyle({
-      color: '#EE495C',
-      fillColor: '#EE495C',
-      fillOpacity: '1',
-    });
-  }
-
-  clearOpacity(e) {
-    const layer = e.target;
-    layer.setStyle({
-      color: '#EE495C',
-      fillColor: '#EE495C',
-      fillOpacity: '0.15',
-    });
-  }
-
-  onEachFeature(feature, layer) {
-    layer.on({
-      click: this.openPanel.bind(this),
-      mouseover: this.setOpacity.bind(this),
-      mouseout: this.clearOpacity.bind(this),
-    });
-  }
-
-  render() {
-    return (
-      <div>
-        {!isEmpty(this.props.fmasData) ? (
-          <LeafletMap
-            center={portland}
-            zoom={11}
-            height={'75vh'}
-            width={'100%'}
-          >
-            <LayerGroup>
-              <GeoJSON
-                style={{
-                  color: '#EE495C',
-                  fillColor: '#EE495C',
-                  fillOpacity: '0.15',
-                }}
-                data={this.props.fmasData}
-                onEachFeature={this.onEachFeature}
-              />
-            </LayerGroup>
-          </LeafletMap>
-        ) : (
-          <h1>Waiting for map to load...</h1>
-        )}
-
-        {this.props.fmaPanelData && !isEmpty(this.props.fmaPanelData) ? (
-          <MapPanel />
-        ) : (
-          <h6>Click on an FMA to view some statistics</h6>
-        )}
-      </div>
-    );
-  }
+    getFmasThunk,
+    getFmasData,
+    renderFmaPanelProperties,
+    getFmaPanelData,
+} from '../../state'
+import MapPanel from '../MapPanel/index'
+
+class FmaMap extends React.Component {
+    componentDidMount() {
+        const { getFmas } = this.props
+        getFmas()
+    }
+
+    render() {
+        const { fmasData, fmaPanelData, renderPanel } = this.props
+
+        return (
+            <div>
+                {fmasData && (
+                    <BaseMap>
+                        <MapOverlay
+                            data={fmasData}
+                            opacity={0.1}
+                            filled
+                            stroked
+                            getPosition={f => f.geometry.coordinates}
+                            onLayerClick={info =>
+                                renderPanel(info.object.properties)
+                            }
+                            getElevation={f => 100}
+                            getFillColor={f => [0, 100, 255, 255]}
+                            getLineColor={f => [220, 69, 86, 255]}
+                        />
+                    </BaseMap>
+                )}
+                {fmaPanelData && !isEmpty(fmaPanelData) ? (
+                    <MapPanel />
+                ) : (
+                    <h6>Click on an FMA to view some statistics</h6>
+                )}
+            </div>
+        )
+    }
```

Comment:
This is four space indented and I hate it 💃 

Can't wait for the impending prettier + eslint PR after the upgrade one. My vote is still for two space indentation, but maybe we should take it to a poll.

Reply:
I also can't wait.